flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_ATSU.js

```diff
@@ -305,6 +305,10 @@ const addWaypointAsync = (fix, mcdu, routeIdent, via) => {
     }
 };
 
+const simBriefToMsfsNatFix = (simBriefFix) => {
```

Comment:
not that this is wrong, just wondering why we use a constant arrow function combination instead of declaring a function right away.

```js
function simBriefToMsfsNatFix(simBriefFix) {}
```

Reply:
As said below, why do we need a function anyways as there's only a single usage.

Reply:
There was some discussion a while ago an arrow functions were preferred. I prefer conventional functions myself, like all other languages. 😆